Parsers/ASimNetworkSession/ProductParsers/vimNetworkSessionMicrosoft365Defender.yaml

```diff
@@ -83,9 +83,9 @@ ParserQuery: |
        SrcUsername = User,
        SrcDvcId = DvcId,
        SrcDvcIdType = 'MDEid',
-       SrcUserIdType = "SID"
+       SrcUserIdType = "SID",
+      DstHostname = tostring(UrlHostname)
```

Comment:
I wonder. How can UrlHostname not be a string?

Reply:
sometimes after parsing some fields remain "dynamic" and to "string", so this is for extra caution